frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -40,6 +55,22 @@ export const EmailFormSection = ({
 
   const { mutateFormEmails } = useMutateFormSettings()
 
+  const isFormPublic = useMemo(
+    () => settings.status === FormStatus.Public,
+    [settings],
```

Comment:
Generally I would personally avoid using `useMemo` for trivial comparisons.

`useMemo` internally does:
1. a comparison: to determine if the internal function should execute)
2. store record of the evaluated value for comparison

A straightforward evaluation of `const isFormPublic = settings.status === FormStatus.Public` only requires a comparison.

Reply:
Ah I see, that makes alot of sense, what I did is definitely redundant, will remove `useMemo`. Thanks!

I noticed a similar usage here by Kar rui, so I thought there might be some react stuff happening under the hood that I was unaware of. Will change this as well.
https://github.com/opengovsg/FormSG/blob/1e66a8aad7d7700da3b6b080129f87428840b3ea/frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSection.tsx#L67